weblate/static/loader-codemirror.js

```diff
@@ -0,0 +1,76 @@
+(function (CodeMirror) {
+
+
+    function getUserList(usernameSearch) {
+        var userList = [];
+        $.ajax({
+            type: 'GET',
+            url: `/api/users/?username=${usernameSearch}`,
+            dataType: 'json',
+            async: false,
```

Comment:
Does it really have to be synchronous? It will IMHO destroy user experience when server is slow to respond.

Reply:
It can be asynchronous, but then the previous response might be rendered while the next request is being performed. Will look into this.